### PR TITLE
[a11y] com_ templates

### PR DESCRIPTION
**Pull request by brianteeman**
Continuing the work to prevent assistive technology reading out the value of an icon. This PR addresses the icons in com_templates as show in the screenshots below.

screenshotr20-43-20
screenshotr20-45-31
screenshotr20-48-19

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9167bd973f9f39993fd855945cf5c57aa4170376<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15061">issues.joomla.org/tracker/joomla-cms/15061</a>.</sub>

File: administrator/components/com_templates/views/styles/tmpl/default.php [hunk 1]
```diff
@@ -78,11 +78,11 @@
 						<td>
 							<?php if ($this->preview && $item->client_id == '0') : ?>
 								<a target="_blank" href="<?php echo JUri::root() . 'index.php?tp=1&templateStyle=' . (int) $item->id ?>" class="jgrid">
-								<span class="icon-eye-open hasTooltip" title="<?php echo JHtml::_('tooltipText', JText::_('COM_TEMPLATES_TEMPLATE_PREVIEW'), $item->title, 0); ?>" ></span></a>
+								<span class="icon-eye-open hasTooltip" aria-hidden="true" title="<?php echo JHtml::_('tooltipText', JText::_('COM_TEMPLATES_TEMPLATE_PREVIEW'), $item->title, 0); ?>" ></span></a>
```
Comment (Quy) [inline administrator/components/com_templates/views/styles/tmpl/default.php]: Remove space before ` ></span>`. Repeat for the other 2 changes.

File: administrator/components/com_templates/views/styles/tmpl/default.php [hunk 2]
```diff
@@ -78,11 +78,11 @@
 						<td>
 							<?php if ($this->preview && $item->client_id == '0') : ?>
 								<a target="_blank" href="<?php echo JUri::root() . 'index.php?tp=1&templateStyle=' . (int) $item->id ?>" class="jgrid">
-								<span class="icon-eye-open hasTooltip" title="<?php echo JHtml::_('tooltipText', JText::_('COM_TEMPLATES_TEMPLATE_PREVIEW'), $item->title, 0); ?>" ></span></a>
+								<span class="icon-eye-open hasTooltip" aria-hidden="true" title="<?php echo JHtml::_('tooltipText', JText::_('COM_TEMPLATES_TEMPLATE_PREVIEW'), $item->title, 0); ?>"></span></a>
```
Comment (wilsonge) [inline administrator/components/com_templates/views/styles/tmpl/default.php]: Given this one is a link do we want to provide something for the screenreader as alt text?

Comment (brianteeman) [inline administrator/components/com_templates/views/styles/tmpl/default.php]: I thought about it but then decided against it when I thought about what the link does - but I was wrong as I thought the link opened the preview image and not the entire site with the module positions. I will update it now. Good spot

Comment (brianteeman) [inline administrator/components/com_templates/views/styles/tmpl/default.php]: done

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 9cb20301ce37d5a5bc7947f3414762dde14a33fd<br /><br />Done code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15061">issues.joomla.org/tracker/joomla-cms/15061</a>.</sub>

**Comment by yasirunilan** [comment]
How to test this PR?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15061">issues.joomla.org/tracker/joomla-cms/15061</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 9cb20301ce37d5a5bc7947f3414762dde14a33fd<br /><br />Code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15061">issues.joomla.org/tracker/joomla-cms/15061</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by rdeutz** [comment]
@brianteeman please check the conflicts, thanks

**Comment by brianteeman** [comment]
conflicts resolved

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**